build.cmd

```diff
@@ -3,6 +3,7 @@ setlocal
 
 set _args=%*
 if "%~1"=="-?" set _args=-help
+if "%~1"=="/?" set _args=-help
```

Comment:
Do we also want `/help` and/or `/h` as flags? If so, then we'd need to add their own clauses:

```cmd
if "%~1"=="/help" set _args=-help
```

Reply:
I don't think we want any `/{letter}` abbreviations. I would only support the ones that were considered by @danmosemsft.